src/test/java/com/wikia/webdriver/testcases/mercurytests/old/TOCTests.java

```diff
@@ -167,4 +167,18 @@ public void MercuryTOCTest_005_RedirectionToHeaderFromOtherPage() {
         result
     );
   }
+
+  @Test(groups = "MercuryTOCTest_006")
+  public void MercuryTOCTest_006_TOCPresence_PlacedUnderInfobox() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)